src/3rd_party/CMakeLists.txt

```diff
@@ -283,9 +283,27 @@ if(HMI_DBUS_API)
   add_subdirectory(dbus-cmake)
 endif()
 
+set(BSON_LIBS_DIRECTORY ${3RD_PARTY_INSTALL_PREFIX_ARCH}/lib PARENT_SCOPE)
+set(BSON_INCLUDE_DIRECTORY ${3RD_PARTY_INSTALL_PREFIX}/include PARENT_SCOPE)
+set(EMHASHMAP_INCLUDE_DIRECTORY ${3RD_PARTY_INSTALL_PREFIX}/include PARENT_SCOPE)
+set(EMHASHMAP_LIBS_DIRECTORY ${3RD_PARTY_INSTALL_PREFIX_ARCH}/lib PARENT_SCOPE)
+set(BSON_LIB_SOURCE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bson_c_lib)
+
+include(ExternalProject)
+ExternalProject_Add(libbson
+    GIT_REPOSITORY "http://github.com/smartdevicelink/bson_c_lib.git"
+    GIT_TAG "master"
```

Comment:
@jacobkeeler are you sure that we need always use the latest version of this package? There may be some issues due changes in bson_c_lib master branch